lib/badge-data.spec.js

```diff
@@ -50,7 +50,8 @@ describe('Badge data helpers', function() {
       given('gratipay', { logo: '' }),
       given(undefined, {}),
     ]).expect(undefined);
-    given('gratipay', {}).assert('should be truthy', assert.ok);
+    given('gratipay', {})
+      .assert('should be truthy', v => expect(v).to.have.lengthOf.at.least(1));
```

Comment:
How about `expect(v).to.not.be.empty` instead of a `expect(v).to.have.lengthOf.at.least(1)`?